components/accessRequestForm/index.jsx

```diff
@@ -127,14 +127,16 @@ export default function AccessRequestForm({
   const handleSubmit = async (e) => {
     e.preventDefault();
     if (selectedAccess.length && selectedPurposes.length) {
+      const date = new Date(selectedDate);
+      date.setUTCHours(23, 59, 59, 0);
```

Comment:
@VirginiaBalseiro I just realised one small extra but, selectedDate is not necessarily set here.

So if we want to be able to do the "never expiring" VCs, we need:

```javascript
let expirationDate;
if (selectedDate) {
    expirationDate = new Date(selectedDate);
    expirationDate.setUTCHours(23, 59, 59, 0);
}
```

And below we can remove assignment:
```javascript
          resources: selectedResources,
          expirationDate,
          resourceOwner: session.info.webId,
```